common/pref_names.cc

```diff
@@ -42,6 +42,7 @@ const char kReferralCheckedForPromoCodeFile[] =
     "brave.referral.checked_for_promo_code_file";
 const char kHTTPSEVerywhereControlType[] = "brave.https_everywhere_default";
 const char kNoScriptControlType[] = "brave.no_script_default";
+const char kAdvancedViewControlType[] = "brave.advanced_view_default";
```

Comment:
I just realized I don't think this constant name or string name for the Preference is specific enough, since this Preference is specific to shields. A better alternative is `kShieldsAdvancedViewEnabled[] = "brave.shields.advanced_view_enabled"`